boards/DIM/Test/Src/Test_StateMachine.cpp

```diff
@@ -239,128 +239,129 @@ TEST_F(DimStateMachineTest, check_drive_state_is_broadcasted_over_can)
+//    // should always be true.
+//    //    App_SharedErrorTable_SetError(error_table, (enum ErrorId)(10), true);
+//    //    App_SharedErrorTable_SetError(error_table, (enum ErrorId)(11), true);
+//
+//    // When an error ID shows up on the 7-segment displays, it will have an
+//    // offset of 500 added to it. This is why we are asserting for the value
+//    // of 10 + 500 = 510 below. The same applies for the rest of this test case.
+//    LetTimePass(state_machine, 999);
+//    ASSERT_EQ(true, set_left_hex_digit_fake.arg0_val.enabled);
+//    ASSERT_EQ(true, set_middle_hex_digit_fake.arg0_val.enabled);
+//    ASSERT_EQ(true, set_right_hex_digit_fake.arg0_val.enabled);
+//    ASSERT_EQ(0, set_left_hex_digit_fake.arg0_val.value);
+//    ASSERT_EQ(1, set_middle_hex_digit_fake.arg0_val.value);
+//    ASSERT_EQ(5, set_right_hex_digit_fake.arg0_val.value);
+//
+//    LetTimePass(state_machine, 1000);
+//    ASSERT_EQ(true, set_left_hex_digit_fake.arg0_val.enabled);
+//    ASSERT_EQ(true, set_middle_hex_digit_fake.arg0_val.enabled);
+//    ASSERT_EQ(true, set_right_hex_digit_fake.arg0_val.enabled);
+//    ASSERT_EQ(1, set_left_hex_digit_fake.arg0_val.value);
+//    ASSERT_EQ(1, set_middle_hex_digit_fake.arg0_val.value);
+//    ASSERT_EQ(5, set_right_hex_digit_fake.arg0_val.value);
+//
+//    LetTimePass(state_machine, 1000);
+//    ASSERT_EQ(true, set_left_hex_digit_fake.arg0_val.enabled);
+//    ASSERT_EQ(true, set_middle_hex_digit_fake.arg0_val.enabled);
+//    ASSERT_EQ(true, set_right_hex_digit_fake.arg0_val.enabled);
+//    ASSERT_EQ(0, set_left_hex_digit_fake.arg0_val.value);
+//    ASSERT_EQ(1, set_middle_hex_digit_fake.arg0_val.value);
+//    ASSERT_EQ(5, set_right_hex_digit_fake.arg0_val.value);
+//
+//    LetTimePass(state_machine, 1000);
+//    ASSERT_EQ(true, set_left_hex_digit_fake.arg0_val.enabled);
+//    ASSERT_EQ(true, set_middle_hex_digit_fake.arg0_val.enabled);
+//    ASSERT_EQ(true, set_right_hex_digit_fake.arg0_val.enabled);
+//    ASSERT_EQ(1, set_left_hex_digit_fake.arg0_val.value);
+//    ASSERT_EQ(1, set_middle_hex_digit_fake.arg0_val.value);
+//    ASSERT_EQ(5, set_right_hex_digit_fake.arg0_val.value);
+//}
 
 // DIM-4
 TEST_F(DimStateMachineTest, check_start_switch_is_broadcasted_over_can_in_drive_state)
 {
     start_switch_is_turned_on_fake.return_val = false;
     LetTimePass(state_machine, 10);
-    //    ASSERT_EQ(CANMSGS_DIM_SWITCHES_START_SWITCH_OFF_CHOICE,
-    //    App_CanTx_GetPeriodicSignal_START_SWITCH(can_tx_interface));
+    ASSERT_EQ(SWITCH_OFF, App_CanTx_DIM_Switches_StartSwitch_Get());
 
     start_switch_is_turned_on_fake.return_val = true;
     LetTimePass(state_machine, 10);
-    //    ASSERT_EQ(CANMSGS_DIM_SWITCHES_START_SWITCH_ON_CHOICE,
-    //    App_CanTx_GetPeriodicSignal_START_SWITCH(can_tx_interface));
+    ASSERT_EQ(SWITCH_ON, App_CanTx_DIM_Switches_StartSwitch_Get());
 }
 
 // DIM-4
 TEST_F(DimStateMachineTest, check_traction_control_switch_is_broadcasted_over_can_in_drive_state)
```

Comment:
These switches seem to have been removed, I'm assuming this is part of the torque vectoring plan? What should be done with these tests